src/Shell/Helper/TableHelper.php

```diff
@@ -78,6 +78,10 @@ protected function _rowSeparator($widths)
      */
     protected function _render($row, $widths, $options = [])
     {
+        if (!is_array($row) || count($row) === 0) {
```

Comment:
there was no parameter hint in the original code, and I didn't want to change the function signature.

Now that I just wrote the above comment. I see that's silly. I'll add type hinting to the method and remove `is_array`